pkg/devfile/parser/data/v2/commands_test.go

```diff
@@ -300,3 +301,102 @@ func TestDevfile200_UpdateCommands(t *testing.T) {
 		})
 	}
 }
+
+func TestDeleteCommands(t *testing.T) {
+
+	tests := []struct {
+		name            string
+		commandToDelete string
+		commands        []v1.Command
+		wantCommands    []v1.Command
+		wantErr         bool
+	}{
+		{
+			name:            "Commands that belong to Composite Command",
```

Comment:
rename the test case to avoid confusion, as composite command is not a special case now